README.md

```diff
@@ -18,6 +18,9 @@ Code documentation is automatically generated using Doxygen tool, available at [
 
 ## Build
 
+It is recomanded to clone noos with submodules:
```

Comment:
There is a typo ("recomanded" -> "recommended"). Or maybe, we can say:
"When cloning the no-OS repository, please pass the --recurse-submodules to the git clone command so the included submodules will be also cloned."

Reply:
Maybe we should think a bit more about what to recommend. I have a PR that adds /libraries/stm32/STM32CubeF4 as a submodule from ST github. If we have projects that will use stm32f0 or f1, we will have to add those as well which makes for lots of submodules to add at each no-OS clone. Maybe we should clone no-OS by recursively cloning submodules based on depth somehow so that not all the submodules get cloned the first time, only the critical ones.